touchforms/formplayer/api.py

```diff
@@ -99,8 +99,8 @@ def start_session(self):
         """
         Start a new session based on this configuration
         """
-        return get_response(json.dumps(self.get_touchforms_dict()), 
-                            self.touchforms_url, auth=self.auth)
+
+        return get_response(json.dumps(self.get_touchforms_dict()), auth=self.auth)
```

Comment:
eventually we'll want to just pass in the dict without json.dumps. we're currently doing:
1. dump data to string
2. parse data into dict
3. dump data into string